x/gov/src/keeper.rs

```diff
@@ -132,6 +138,10 @@ impl<
         }
         self.gov_params_keeper.set(ctx, params);
 
+        self.auth_keeper
```

Comment:
This should be a part of gaia? You added a new keeper only for simple creation of account.

Reply:
For now I thin it would be best to stick to how Cosmos do it i.e. inside the module.